### DISCUSSION TITLE
Renaming of the `hypotests` submodule.

### DISCUSSION THREAD

**Discussion by marinang** [scikit-hep/hepstats#32] [Ideas]
Since `hepstats` has rooms for other statistics tools such as tests like:

- Energy test
- Maybe BumpHunter

It would be good to rename the `hypotests` submodule to stress the point that it is a submodule to do hypothesis tests, but also upper limit or confidence interval calculations, using the likelihood ratio as a test statistic.

I have a few ideas like:
- `likelihoodratio`
- `lrstat`

Any suggestion is welcomed.

**Comment by eduardo-rodrigues** [comment]
Hi @marinang. You can find a new name or rather split into several submodules. In any case, if you want more feedback, then this question seems very appropriate for a thread on Gitter ... Just my 2 cents.

**Comment by marinang** [comment]
Thanks @eduardo-rodrigues I will post something 👍
